packages/web-twig/src/Resources/components/Tooltip/README.md

```diff
@@ -139,18 +139,20 @@ Advanced floating functionality is provided by JavaScript plugin and by [Floatin
-| `placement`                     | [Placement Dictionary][dictionary-placement] | "bottom" | ✕        | Placement of tooltip                                                                                                                                                                                                                                                       |
+| Attribute                       | Type                                         | Default        | Required | Description                                                                                                                                                                                                                                                                |
+| ------------------------------- | -------------------------------------------- | -------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `closeLabel`                    | `string`                                     | `Close`        | ✕        | Close label                                                                                                                                                                                                                                                                |
+| `enableFlipping`                | `bool`                                       | true           | ✕        | Enables [flipping][floating-ui-flip] of the element’s placement when it starts to overflow its boundary area. For example `top` can be flipped to `bottom`.                                                                                                                |
+| `enableFlippingCrossAxis`       | `bool`                                       | true           | ✕        | Enables flipping on the [cross axis][floating-ui-flip-cross-axis], the axis perpendicular to main axis. For example `top-end` can be flipped to the `top-start`.                                                                                                           |
+| `enableHover`                   | `bool`                                       | false          | ✕        | Enables hover functionality when the cursor is positioned over the trigger.                                                                                                                                                                                                |
```

Comment:
Is this prop needed when we are using the `triggers`?

Reply:
no, i think i failed rebase... removed, thank you